src/main/java/reposense/commits/CommitInfoAnalyzer.java

```diff
@@ -29,8 +32,9 @@
     private static final Logger logger = LogsManager.getLogger(CommitInfoAnalyzer.class);
     private static final String MESSAGE_START_ANALYZING_COMMIT_INFO = "Analyzing commits info for %s (%s)...";
 
-    private static final String LOG_SPLITTER = "\\|\\n\\|";
+    private static final String LOG_SPLITTER = "\\?\\n\\?";
```

Comment:
Need advice here, I am switching the delimiter to be `?` instead of `|`, this is because the `git log ... --stat` returns the files changed in the following format:
`<file path>    | <num of lines changed>`,
which overlaps if we stick with using `|` as a delimiter.

Don't think there's a way to change the format of the `git log ... --stat`, which is why i decided to change the delimiter.